src/Future.re

```diff
@@ -1,17 +1,47 @@
 type getFn('a) = ('a => unit) => unit;
 
+type executorOptions = [ | `none | `trampoline];
```

Comment:
Maybe rename to `executorType`?

Reply:
I think `trampoline` is good.  If you run into the stack problem, `trampoline` is something you'll definitely find right away.